src/App.js

```diff
@@ -1,55 +1,19 @@
-import logo from './logo.svg';
-import { useEffect, useState } from 'react';
+
 import './App.css';
+import TodoList from './TodoList';
+import AddTodoForm from './addTodoForm';
 
 function App() {
-  const [todoList, setTodoList] = useState([{
-    id: 1,
-    title: 'complete assignment'
-  },
-  {
-    id: 2,
-    title: 'push assignment to Github'
-  }])
-
-  const [inputText, setInputText] = useState('')
-
-  useEffect(() => {
-    console.log(inputText)
-  }, [inputText])
-
-  useEffect(() => {
-    console.log(todoList)
-  }, [todoList])
-
-  const textChange = (e) => {
-    setInputText(e.target.value)
-  }
-
-  const submitItem = (e) => {
-
-    let newList = todoList;
-    newList.push({ id: todoList.length + 1, title: inputText })
-    setTodoList(newList)
-    setInputText('')
-    e.target.previousElementSibling.value = ''
-  }
-
   return (
     <div>
       <h1>Todo List</h1>
-      <label htmlFor='search'>submit next to do item:</label>
-      <input id='todoItem' onChange={textChange} type='text' />
-      <button id='submit' onClick={submitItem}>Submit</button>
+      <AddTodoForm />
+
+      <ul>
+        <TodoList />
+      </ul>
 
-      <div>
-        <ul>
-          {todoList.map((item) => {
-            console.log(item)
-            return <li id={item.id - 1}>{item.id} {item.title}</li>
-          })}
-        </ul>
-      </div>
+      <label htmlFor='newItem'>submit next to do item:</label>
```

Comment:
Nitpick:
Typically, label elements should only exists within form elements. That said, they're primarily used to provide a description for a form control element or provide a large clickable area for a form control element (like an input). See this [label documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label) for more info.